packages/asc-ui/src/components/Checkbox/Checkbox.test.tsx

```diff
@@ -62,4 +62,17 @@ describe('Checkbox', () => {
 
     expect(icon).not.toHaveStyleRule('background-color', expect.any(String))
   })
+
+  it('should handle refs', () => {
+    const ref = React.createRef<HTMLInputElement>()
+
+    rerender(
```

Comment:
I believe we can use `render` here instead of `rerender`

Reply:
that will result in code doubling

Reply:
@jasperswart what do you mean?